acquia_lift.admin.unibar.inc

```diff
@@ -209,6 +209,7 @@ function acquia_lift_element_variation_details_form_submit($form, &$form_state)
 
     module_load_include('inc', 'personalize_elements', 'personalize_elements.admin');
     $option_set = personalize_elements_convert_form_values_to_option_set($values);
+    $option_set->preview_link = $values['pages'];
```

Comment:
Not at that point.  This is only when creating a new personalize elements variation set from the unibar.  The unibar populates the pages field with the current url using JavaScript.  The user can't touch it.